server/data/prisonerContactRegistryApiClient.ts

```diff
@@ -15,16 +14,12 @@ class PrisonerContactRegistryApiClient {
   constructor(private readonly restclient: RestClient) {}
 
   getPrisonerSocialContacts(offenderNo: string): Promise<Contact[]> {
-    if (config.apis.prisonerContactRegistry.enabled) {
```

Comment:
figured I may as well remove the toggle and test data - though could leave it in place if you prefer?

Reply:
Happy to remove 👍